manifests/base/transformers/mongoDbUri/selectivepatch.yaml

```diff
@@ -24,6 +24,31 @@ patches:
                 secretKeyRef:
                   key: mongoDbUri
                   name: secrets
+- target:
+    kind: Deployment
+  patch: |-
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: component
+    spec:
+      template:
+        spec:
+          volumes:
+          - name: qliksense-secrets
+            secret:
+              secretName: secrets
+          containers:
+          - name: main
+            volumeMounts:
+            - mountPath: /run/secrets/qlik.com/qliksense
+              name: qliksense-secrets
+              readOnly: true
+            env:
+            - $patch: delete
+              name: MONGODB_URI_FILE
+            - name: MONGODB_URI_FILE
+              value: /run/secrets/qlik.com/qliksense/mongoDbUri
```

Comment:
well generally resource contract use `mongodbUri` for this key but won't harm to keep it like that as this is the qliksense secret anyways